.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -2,10 +2,16 @@ name: Scanner dev vulnerability update
 on:
   schedule:
   - cron: "30 */4 * * *"
+  push:
+    branches:
+    - yli3/*
 
 jobs:
   upload-dev-vulnerabilities:
     runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.61
+      options: -v /tmp:/tmp # make sure Claicore blob can access tmp folder
```

Comment:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontainervolumes

Reply:
I saw some discussion online says `volumes:` may not work but `options: -v /tmp:/tmp` should work the same

Reply:
> I saw some discussion online says `volumes:` may not work but `options: -v /tmp:/tmp` should work the same

interesting, do you have a link to this?

Reply:
> I saw some discussion online says volumes: may not work but options: -v /tmp:/tmp should work the same

I would say `volumes` is generic specification of volumes, while `options` is specific to the container runtime; But, yeah, can you show us the discussion in question?

Reply:
Did you experience the same issue? I am wondering if this is still relevant four years later.

Reply:
Yeah, I will try the `volumes`